documentation/package.json

```diff
@@ -50,7 +50,7 @@
     "react-dom": "^16.8.4",
     "react-live": "^2.3.0",
     "styled-components": "^5.1.1",
-    "three": "0.125.2"
+    "three": "0.135.0"
```

Comment:
If this works, this is a good idea - but do we know whether it works? The reason it is stuck on 125, is that the Reveal 1.x documentation depends on it. Have we tested whether they still work with this change?